test/kitchen/Berksfile

```diff
@@ -1,6 +1,6 @@
 source 'https://supermarket.chef.io'
 
-cookbook 'datadog', '~> 3.3.0'
+cookbook 'datadog', git: "https://github.com/DataDog/chef-datadog", branch: "albertvaka/4.0-refactor"
```

Comment:
You might want to pin a commit instead of a branch, when I make changes to that branch I don't want to break this if I push a bad commit.